.nvmrc

```diff
@@ -1,2 +1 @@
-4
-
+6.9.1
```

Comment:
But it means that we always need to have (and switch to) 6.9.1 locally, even when 6.9.2+ comes out 😞